ci/do_ci.sh

```diff
@@ -154,6 +154,14 @@ elif [[ "$CI_TARGET" == "bazel.gcc" ]]; then
   echo "bazel release build with gcc..."
   bazel_binary_build release
   exit 0
+elif [[ "$CI_TARGET" == "bazel.gcc.cross_aarch64" ]]; then
+  BAZEL_BUILD_OPTIONS+="--crosstool_top=@envoy_build_tools//toolchains/configs/linux/gcc/bazel_3.4.1/cc:toolchain"
```

Comment:
When I started on this, I learnt that `--crosstool_top` is a deprecated thing, and `--toolchain` is encouraged. Wa this explicitly chose due to some Envoy-specific reason?

Reply:
It seems like `--platforms` is encouraged instead of `--crosstool_top` - https://docs.bazel.build/versions/master/platforms-intro.html#goal

I will change that. Thanks for pointing that out!